### DISCUSSION TITLE
Adding the option to set quota for users

### DISCUSSION THREAD

**Discussion by ghost** [filebrowser/filebrowser#1550] [Ideas]
Hello,

**Is your feature request related to a problem? Please describe.**
I saw in #711 a feature request about adding to option to set a quota per users in the application, but it was closed at the time of the question because of a future archiving of the repository. 

**Describe the solution you'd like**
Do you think it is possible, for the administrator, to be able to set a global quota per user plus being able to edit it case by case if wanted on the specific user panel?

Thanks in advance for your opinion on it!

**Comment by dillfrescott** [comment]
I would love this feature!
